pkgs/top-level/metrics.nix

```diff
@@ -49,6 +49,9 @@ runCommand "nixpkgs-metrics"
     run nix-env.qa nix-env -f ${nixpkgs} -qa
     run nix-env.qaDrv nix-env -f ${nixpkgs} -qa --drv-path --meta --xml
 
+    num=$(nix-env -f ${nixpkgs} -qa --drv-path --meta --xml | grep '<item ' | wc -l)
+    echo "nix-env.qaDrv.number $num" >> $out/nix-support/hydra-metrics
```

Comment:
How about just `nix-env -f ${nixpkgs} -qa | wc -l`

Also, I'd prefer the name "count" instead of "number"